src/Command/Project/ProjectCreateCommand.php

```diff
@@ -60,9 +63,24 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $options = $this->form->resolveOptions($input, $output, $questionHelper);
 
+        if ($input->getOption('template') !== false) {
+            if (empty(parse_url($input->getOption('template'), PHP_URL_PATH))) {
+                $temp_provided = true;
+            }
+            else {
+                $temp_provided = false;
+            }
+            $this->template_form = Form::fromArray($this->getTemplateFields($temp_provided));
```

Comment:
field created dynamically, maybe add it as a `protected $template_form` to the class?

Reply:
good idea.